LICENSE-BSD

```diff
@@ -0,0 +1,24 @@
+Copyright (c) 2015, Linaro Limited
+All rights reserved.
```

Comment:
Please drop this line, it's not part of the license.

Reply:
My apologies Jens - in applying this fix I removed the first and the second line. The second line here is not part of the license, but the first is. I've uploaded a fix as #752